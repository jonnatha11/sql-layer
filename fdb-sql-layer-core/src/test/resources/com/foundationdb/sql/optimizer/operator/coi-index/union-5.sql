select cid, name from customers
UNION ALL
select cid, cast (order_date as VARCHAR(32)) from orders