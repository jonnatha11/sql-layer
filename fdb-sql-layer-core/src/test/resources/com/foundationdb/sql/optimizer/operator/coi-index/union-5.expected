PhysicalSelect[cid:int, name:varchar(32)]
  UnionAll_Default()
    Project_Default(customers.cid, CAST(customers.name AS VARCHAR(32)))
      IndexScan_Default(Index(customers.name), name, cid)
    Project_Default(orders.cid, CAST(orders.order_date AS VARCHAR(32)))
      IndexScan_Default(Index(orders.order_date), order_date, cid, oid)